src/plugins/irc-events/link.js

```diff
@@ -49,7 +49,10 @@ function parse(msg, url, res, client) {
 	case "text/html":
 		var $ = cheerio.load(res.text);
 		toggle.type = "link";
-		toggle.head = $("title").text();
+		toggle.head =
+			$("meta[property=\"og:title\"]").attr("content")
+			|| $("title").text()
+			|| "No title found.";
```

Comment:
Let's instead refuse to show info if there is no preview image/title/description? Or only show "Untitled page" if there is an image. This will need to done in a separate PR though.